### ISSUE TITLE
Built-in's in solid version are being unintentionally published as preview

### ISSUE THREAD

**Issue by alvsan09** [eclipse-theia/vscode-builtin-extensions#105]
When publishing buil-in extensions having a version that  includes the 'next' word on it, 
they are supposed to be marked as 'preview'  which allow consumers to easily identify them before consuming them

however as it can be seen in the picture; Extensions with solid revisions have been already published with the 'preview' flag set to 'true'

There are two sides of this issue: 
1) Correct the logic, so 'next' versions are the only ones marked as 'preview' 
2) Investigate how to correct this in the data base.  

![Screenshot from 2022-12-14 12-51-53](https://user-images.githubusercontent.com/76971376/207670958-1514eb73-eaec-4d41-aed7-12b1762f56a1.png).

**Comment by CiciDongxue** [comment]
您好，我已收到您的来信，暂时无法回复您的邮件，望见谅。

**Comment by marcdumais-work** [comment]
Alvaro, as discussed, we currently include the package.json `preview` flag only for preview (next) extensions:
> "preview": true

We can try including it in the future, but set to `false` for solid revisions and see if it helps. 

For what's already published, it seems openvsx would need to revisit their interpretation of what it means when the flag is not present. 

Examples of the current situation:

No `preview` flag is set in package.json, for solid versions. e.g.:
https://open-vsx.org/api/vscode/bat/1.62.3/file/vscode.bat-1.62.3.vsix

The `preview  flag is set (to false) for `next` versions:
https://open-vsx.org/api/vscode/bat/1.63.0-next.66b1668b667/file/vscode.bat-1.63.0-next.66b1668b667.vsix

**Comment by amvanbaren** [comment]
Hi @marcdumais-work, I think you want to use the `--pre-release`[[1](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#prerelease-extensions)] flag for `next` versions. It used to be that `preview`[[2](https://devblogs.microsoft.com/visualstudio/new-preview-label-for-visual-studio-extensions-2/)] had the functionality of `pre-release`, but now these flags work the same as on the MS marketplace.

In short, `preview` is an indicator to a user that the extension (all versions) is not yet feature complete and may contain bugs. While `pre-release` lets a user regularly get the latest extension version before the official extension release.
